src/middleware/__tests__/addUserContext.int.test.ts

```diff
@@ -28,14 +34,22 @@ describe('requireAuthentication', () => {
 		} as unknown as AuthenticatedRequest;
 		const mockResponse = {} as unknown as Response;
 
-		const runAssertions = async (err: unknown) => {
-			expect(err).toBe(undefined);
-			const user = await loadUserByAuthenticationId('foo@example.com');
-			expect(mockRequest.user).toEqual(user);
-		};
+		loadTableMetrics('users')
```

Comment:
Ah I see, this choice becomes clearer once we measure for offset(now that we are expecting baselineUserCount to be offset by 1)